docs/program-setup/serial/serial_communication.md

```diff
@@ -6,57 +6,67 @@ The Arm Mbed microcontroller on your board can communicate with a host PC over t
+[![View code](https://www.mbed.com/embed/?url=https://github.com/ARMmbed/mbed-os-examples-docs_only/blob/master/Tutorials_SerialComm/Serial_STDOUT/)](https://github.com/ARMmbed/mbed-os-examples-docs_only/blob/master/Tutorials_SerialComm/Serial_STDOUT/main.cpp)
 
-## Using terminal applications
+### Viewing output in a serial terminal
 
-Terminal applications run on your host PC. They provide a window where your Mbed board can print and where you can type characters back to your board.
+Serial terminals run on your host PC. They provide an interface where your Mbed board can print and where you can type characters back to your board.
 
-<span class="tips">**Serial configuration:** The standard setup for the USB serial port is 9600 baud, 8 bits, 1 stop bit, no parity (9600-8-N-1)</span>
+Mbed CLI provides a serial terminal which is configured with a default baud rate of `9600`. When a single board is connected, simply run: 
 
-### Installing an application for Windows
+```
+mbed sterm
+```
 
-There are many terminal applications for Windows, including:
+(If you have multiple boards connected, please refer to [Additional options of mbed sterm](#additional-options-of-mbed-sterm).)
 
-- [CoolTerm](http://freeware.the-meiers.org/) - this is the application we use in this example. We use it often because it usually "just works".
-- [Tera Term](http://sourceforge.jp/projects/ttssh2/files).
-- [PuTTY](http://www.chiark.greenend.org.uk/~sgtatham/putty/).
-- Some Windows PCs come with **Hyperterminal** installed.
+Messages printed after this point will be displayed, restart the application using the board's reset button or press `Ctrl + B` on the serial terminal.
+The console prints "Hello World!" to the terminal after the reset.
 
-### Configuring the connection
+To exit the serial terminal, press Ctrl + C.
 
-1. Plug in your Mbed board.
-1. Open CoolTerm.
-1. Click **Connect**. This opens up an 8-n-1 9600 baud connection to the first available serial port. If you have more than one board plugged in, you may need to change the port under **Options > Serial Port > Port**.
+<span class="tips">**Tip:** To compile, flash and open a serial terminal in one command line, you can append `--flash --sterm` to `mbed compile`.</span>
 
-Check your connection parameters:
+## Additional options of mbed sterm
 
-1. Select **Options > Serial Port**.
-1. You should see 9600 baud, 8 bits, 1 stop bit, no parity (9600-8-N-1).
-1. If you do not see your board, click **Re-Scan Peripherals**.
+If you have multiple boards connected, run `mbedls` to find out the port of the board you want to use, then run `mbed sterm` with `-p <PORT>` to specify.
 
-Your terminal program is now configured and connected.
+If your application uses a baud rate, specify with `-b <BAUDRATE>` when opening the serial terminal.
```

Comment:
```suggestion
- If your application uses a baud rate, specify it with `-b <BAUDRATE>` when opening the serial terminal.
```

Reply:
Don't they always use baud rate? Is it just if it's not the default set somewhere? Shouldn't it match the board?

Reply:
Will fix it